### DISCUSSION TITLE
iOS NFC连接至Yubico Authenticator时无法显示所有TOTP Key

### DISCUSSION THREAD

**Discussion by samduanx** [canokeys/canokey-product#22] [Q&A] [answered]
问题如题所示。经测试iOS 17.2.1（iPhone 13 Pro，最新版本Authenticator）只能读取7个TOTP Key，在Android 13（Xperia XZ2，最新版本Authenticator）和WebUSB console上能够读取全部Key（11个）。
需要确认问题是App问题还是硬件问题。

**Comment by dangfan** [comment, marked as answer]
请使用 [CanoKey Console](https://apps.apple.com/app/canokey-console/id6476454147)
